engine/server/engine/main.go

```diff
@@ -131,6 +134,40 @@ func runMain() error {
 		},
 	)
 
+	go func() {
+		staticPath := "/run/webapp"
```

Comment:
`staticPath` and i`ndexPath` should be`serverArgs` fields instead

Reply:
not sure what you mean here?

Reply:
that these values can be engine's arguments, like the `serverArgs.GrpcListenPortNum` but we can do it in a next PR